operator/controllers/kyma_controller_helper_test.go

```diff
@@ -119,9 +119,7 @@ func SKRModuleExistWithOverwrites(kymaName string, moduleTemplate *v1alpha1.Modu
 
 func getModule(kymaName string, moduleTemplate *v1alpha1.ModuleTemplate) (*unstructured.Unstructured, error) {
 	component := moduleTemplate.Spec.Data.DeepCopy()
-	if moduleTemplate.Spec.Target == v1alpha1.TargetRemote {
-		component.SetKind("Manifest")
-	}
+	component.SetKind("Manifest")
```

Comment:
since we changed the manifest-operator to module-manager, maybe better to define it as a const and give an explainable name (I don't have a good name candidate yet, something like `ManagedModuleKind`), it's hard to understand this Manifest for others if don't know the history I think.

Reply:
Since we plan to refactor this to make use of the real type instead of unstructured, I think this is only temporary and we can leave it in